collectives/templates/index.html

```diff
@@ -48,19 +48,21 @@ <h3>
     <div id="filters">
         <div class="">
             Afficher aussi les collectives:
-            <label> passées <input
+            <label for="pastcheckbox"> passées </label>
```

Comment:
Modification plutôt inutile: on a pas besoin de lier les labels par nom si l'input est dedans.

Reply:
Faux, ca ne marche pas sous la plupart des browsers

Reply:
Purée, ca fait ch*** les navigateurs qui ne suivent pas les normes!

https://www.w3.org/TR/html52/sec-forms.html#the-label-element

> The label element represents a caption in a user interface. The caption can be associated with a specific form control, known as the label element’s labeled control, either using the for attribute, or by putting the form control inside the label element itself.

Dans ce cas, ok pour la modif, mais autant lui faire une PR dédiée alors

Reply:
Ah ben non, c'est moi qui raconte n'importe quoi, pas moyen de reproduire chez moi... je retesterai au taf demain, pour l'instant je vais juste pousser le reste